solid/lib/Notifications/SolidWebHook.php

```diff
@@ -0,0 +1,57 @@
+<?php
+    namespace OCA\Solid\Notifications;
+
+    class SolidWebhook implements SolidNotificationsInterface
+    {
+        public function send($path, $type) {
+            $subscribedUrls = $this->getSubscribedUrls($path, $type);
+            foreach ($subscribedUrls as $url) {
+                $this->postUpdate($url, $path, $type);
+            }
+        }
+        private function getSubscribedUrls($path, $type) {
+	    return []; // FIXME: Read this from the subscriptions
+        }
+        private function postUpdate($url, $path, $type) {
+            try {
+                $postData = $this->createUpdatePayload($path, $type);
+                $opts = array('http' =>
+                    array(
+                        'method'  => 'POST',
+                        'header'  => 'Content-Type: application/ld+json',
+                        'content' => $postData
+                    )
+                );
+                $context  = stream_context_create($opts);
+                $result = file_get_contents($webhook, false, $context);
```

Comment:
Where is `$webhook` coming from? Should that be `$url`?

Reply:
yes - renamed them all to webhookUrl to be more clear.